server/routes/approval.js

```diff
@@ -20,7 +18,7 @@ module.exports = ({licenceService, prisonerService}) => (router, audited) => {
 
             const {nextPath, pageDataMap} = formConfig[formName];
             const dataPath = pageDataMap || ['licence', 'approval', 'release'];
-            const data = getIn(res.locals.licence, dataPath) || {};
+            const data = firstItem(req.flash('userInput')) || getIn(res.locals.licence, dataPath) || {};
```

Comment:
what's going on here?

Reply:
If the page is redirected to after a validation error then the data to fill the form fields with will be in req.flash. If it is accessed having already been inputted it will be in the licence. Otherwise empty